roles/lib_openshift/src/ansible/oc_adm_ca_server_cert.py

```diff
@@ -0,0 +1,36 @@
+# pylint: skip-file
+# flake8: noqa
+
+def main():
+    '''
+    ansible oc adm module for ca create-server-cert
+    '''
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            state=dict(default='present', type='str', choices=['present']),
+            debug=dict(default=False, type='bool'),
+            kubeconfig=dict(default='/etc/origin/master/admin.kubeconfig', type='str'),
+            cmd=dict(default=None, require=True, type='str'),
+            # oadm ca create-server-cert [options]
+            cert=dict(default=None, type='str'),
+            key=dict(default=None, type='str'),
+            signer_cert=dict(default=None, type='str'),
+            signer_key=dict(default=None, type='str'),
+            signer_serial=dict(default=None, type='str'),
+            hostnames=dict(default=[], type='list'),
+            overwrite=dict(default=False, type='bool'),
```

Comment:
I think it would be useful. Not just for this module, but for others as well. 

A common use case that I'm seeing is where there are two tasks back-to-back to do `state: present` and then `state: list` to retrieve the object.